addon/-private/helpers.js

```diff
@@ -311,3 +315,12 @@ export function getProperty(object, pathToProp) {
 }
 
 export const assign = Ember.assign || Ember.merge;
+
+export function wait() {
+  const { settled, wait } = emberTestHelpers;
```

Comment:
I think it will fail if `require.has('ember-test-helpers') === false`

Reply:
Ha, good point. I'll go back to my original version:
```
if (emberTestHelpers) {
  ...
}
```

Reply:
Still trying to figure out how to write tests for this.